frontend/src/views/applications/EligibilityView.vue

```diff
@@ -0,0 +1,284 @@
+<template>
+  <v-form ref="form">
+    <AppAlertBanner type="info">
+      If you answer No to any of these questions, you will not be eligible for OFM funding. More information can be at the
+      <a href="https://www2.gov.bc.ca/gov/content?id=F0D0FD2A00064CDDBF634CF83E4E2599" target="_blank">OFM website</a>
+      for eligibility.
+    </AppAlertBanner>
+    <v-card variant="outlined" class="pa-4 my-6">
+      <div>
+        <div>Is your facility currently in receipt of Child Care Operating Funding or $10 a Day ChildCareBC funding for a minimum of 1 year?</div>
+        <AppYesNoRadioGroup
+          id="greater-one-year-ccof-tdad"
+          v-model="model.greaterOneYearCCOFTDAD"
+          :rules="rules.required"
+          :hide-details="readonly"
+          :disabled="readonly"
+          @update:model-value="validateResponse(model.greaterOneYearCCOFTDAD)" />
+      </div>
+      <div v-if="isCCOFProgram()">
+        <div>Is your facility approved or is your application being processed by the Ministry to participate in the Child Care Fee Reduction Initiative for the current year (if eligible)?</div>
+        <AppYesNoRadioGroup
+          id="ccfri-participation"
+          v-model="model.ccfriParticipation"
+          :rules="rules.required"
+          :hide-details="readonly"
+          :disabled="readonly"
+          @update:model-value="validateResponse(model.ccfriParticipation)" />
+      </div>
+      <div>
+        <div>Is your facility in good standing with the Ministry of Education and Child Care?</div>
+        <AppYesNoRadioGroup
+          id="ministry-good-standing"
+          v-model="model.ministryGoodStanding"
+          :rules="rules.required"
+          :hide-details="readonly"
+          :disabled="readonly"
+          @update:model-value="validateResponse(model.ministryGoodStanding)" />
+      </div>
+      <div>
+        <div>Is your facility and licence in good standing with your regional Health Authority?</div>
+        <AppYesNoRadioGroup
+          id="health-authority-good-standing"
+          v-model="model.healthAuthorityGoodStanding"
+          :rules="rules.required"
+          :hide-details="readonly"
+          :disabled="readonly"
+          @update:model-value="validateResponse(model.healthAuthorityGoodStanding)" />
+      </div>
+      <div>
+        <div>
+          Do all your current Early Childhood Educator(s) have an active, valid certificate? (If your licence does not require an Early Childhood Educator and you have none working in your operation,
```

Comment:
When I try and select a N/A field - my application does not save

![image](https://github.com/user-attachments/assets/ecebd264-18ad-4c47-99b3-5e759dd233b1)

I think this is because N/A is not a value in CRM?